common/lib/xmodule/xmodule/js/src/video/00_sjson.js

```diff
@@ -27,14 +27,24 @@ function() {
             return sjson.text.length;
         };
 
-        var search = function (time) {
-            var start = getStartTimes(),
-                max = size() - 1,
-                min = 0,
-                index;
-
-            if (time < start[min]) {
-                return -1;
+        function search(time, startTime, endTime) {
+            var start;
```

Comment:
Sorry if there was a miscommunication w/r/t var declarations and/or
"minimizing the diff". Tactics like adding each declaration on it's own
line are great for minimizing future conflicts _only_ when they're added
originally. Forking lines of code to do so needlessly increases the
delta from upstream, which is what we want to avoid.

So in this case, at least, if you look at the variable declarations
here, git thinks that every single line is changing. In reality, the
only things that are actually changing are that we're no longer
assigning a value to `start` and we're declaring an additional variable
`results`; we want the diff the convey this idea, as much as possible.

So if we revert/update this like:
```javascript
var results;
var start,
    max = size() - 1,
    min = 0,
    index;
```

The diff goes from:
```diff
-           var start getStartTimes(),
-                max = size() - 1,
-                min = 0,
-                index;
+            var start;
+            var max = size() - 1;
+            var min = 0;
+            var index;
+            var results;
```

to:
```diff
-           var start getStartTimes(),
+            var results;
+            var start,
```

Does that make more sense?